sdk/src/main/java/io/dapr/client/AbstractDaprClient.java

```diff
@@ -72,164 +72,165 @@
    * {@inheritDoc}
    */
   @Override
-  public Mono<Void> publishEvent(String pubsubName, String topic, Object data) {
-    return this.publishEvent(pubsubName, topic, data, null);
+  public Mono<Void> publishEvent(String pubsubName, String topicName, Object data) {
+    return this.publishEvent(pubsubName, topicName, data, null);
   }
 
   /**
    * {@inheritDoc}
    */
   @Override
-  public Mono<Void> publishEvent(String pubsubName, String topic, Object data, Map<String, String> metadata) {
-    PublishEventRequest req = new PublishEventRequestBuilder(pubsubName, topic, data).withMetadata(metadata).build();
+  public Mono<Void> publishEvent(String pubsubName, String topicName, Object data, Map<String, String> metadata) {
+    PublishEventRequest req = new PublishEventRequestBuilder(pubsubName, topicName,
+            data).withMetadata(metadata).build();
     return this.publishEvent(req).then();
   }
 
   /**
    * {@inheritDoc}
    */
-  public <T> Mono<T> invokeService(
+  public <T> Mono<T> invokeMethod(
       String appId,
-      String method,
-      Object request,
+      String methodName,
+      Object data,
       HttpExtension httpExtension,
       Map<String, String> metadata,
       TypeRef<T> type) {
-    InvokeServiceRequestBuilder builder = new InvokeServiceRequestBuilder(appId, method);
+    InvokeServiceRequestBuilder builder = new InvokeServiceRequestBuilder(appId, methodName);
     InvokeServiceRequest req = builder
-        .withBody(request)
+        .withBody(data)
```

Comment:
@artursouza Should `body` be renamed to data?
Also should `InvokeServiceRequest` and associated builder be renamed to `InvokeMethodRequest` since the API methods are being renamed?

Reply:
@artursouza On second thought why is invokeService being renamed to invokeMethod? I think invokeService makes sense since we are invoking a particular method in that service.

Reply:
Let's discuss this with the maintainers and see if we can change it. Let's not merge this PR for this milestone, so we have time to change it.

Reply:
I would say let's do this in a followup PR after discussing with maintainers.

Reply:
Well, we have cut the milestone already, so we can merge this PR soon and still have another PR to rename to invokeService again if maintainers agree to name it back.